modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3384,6 +3384,168 @@ Response::
 s.@max_best_move.deltaQ > 0 THEN\n                        IF -s.@max_best_move.weight < s.@sum_cc_weight THEN   // smallest best_move weight < current weight\n                            s.@cc_list.clear(),\n                            s.@cc_list += s.@max_best_move.cc,\n                            s.@sum_cc_weight = -s.@max_best_move.weight,\n                            @@sum_cc_change += 1\n                        ELSE\n                            IF -s.@max_best_move.weight == s.@sum_cc_weight AND getvid(s.@cc_list.get(0)) < getvid(s.@max_best_move.cc)  THEN\n                                s.@cc_list.clear(),\n                                s.@cc_list += s.@max_best_move.cc,\n                                s.@sum_cc_weight = -s.@max_best_move.weight,\n                                @@sum_cc_change += 1\n                            END\n                        END\n                    END;\n        IF print_info THEN\n            PRINT @@sum_cc_change AS IterChangeCount;\n        END;\n    END;\n    // process node with outdegree=1\n    // follow the vertex to its neighbor community\n    // if the neighbor also have outdegree=1, mark the two vertices as one community\n    Start = {v_type};\n    Start = SELECT s\n            FROM Start:s -(e_type:e)- :t\n            WHERE s.@sum_outdegree == 1 AND t.@sum_outdegree != 1\n            ACCUM\n                s.@cc_list += t.@cc_list.get(0);\n    IF print_info THEN\n        PRINT Start.size() AS VertexFollowedToCommunity;\n    END;\n    Start = {v_type};\n    Start = SELECT s\n            FROM Start:s -(e_type:e)- :t\n            WHERE s.@sum_outdegree == 1 AND t.@sum_outdegree == 1\n            ACCUM\n                IF getvid(s) <= getvid(t) THEN\n                    s.@cc_list += s\n                ELSE\n                    s.@cc_list += t\n                END;\n    IF print_info THEN\n        PRINT Start.size() AS VertexFollowedToVertex;\n    END;\n    // process node with outdegree=0\n    // assign them to communities containing only itself\n    Start = {v_type};\n    Start = SELECT s\n            FROM Start:s\n            WHERE s.@sum_outdegree == 0\n            ACCUM\n                s.@cc_list += s;\n    IF print_info THEN\n        PRINT Start.size() AS VertexAssignedToItself;\n    END;\n    // save result\n    Start = {v_type};\n    Start = SELECT s\n            FROM Start:s\n            POST-ACCUM\n                IF result_attr != \"\" THEN\n                    s.setAttr(result_attr, getvid(s.@cc_list.get(0)))\n                END,\n                IF file_path != \"\" THEN\n                    f.println(s, getvid(s.@cc_list.get(0)))\n                END;\n    // print result satistic\n    IF print_info THEN\n        Start = SELECT s\n                FROM Start:s\n                WHERE s.@cc_list.size() > 0\n                POST-ACCUM\n                    @@community_map += (getvid(s.@cc_list.get(0)) -> 1);\n        PRINT @@community_map.size() AS FinalCommunityCount;\n    END;\n  }",
```

Comment:
I think we can collapse the long query body since there is no significance to show the full template query.
or we can provice an shorter template query here.